.release-it.json

```diff
@@ -6,13 +6,12 @@
 	"pkgFiles": ["package.json", "package-lock.json"],
 	"increment": "conventional:angular",
 	"scripts": {
-		"afterBump": "npm run check:nightly:stark-versions -- ${version} && npm run generate:changelog",
 		"afterRelease": false,
-		"beforeStage": false,
-		"beforeStart": "npm run check:starter:stark-versions",
-		"changelog": "npm run generate:changelog-recent"
+		"beforeStage": "npm run check:nightly:stark-versions -- ${version} && npm run generate:changelog",
+		"beforeStart": "npm run check:starter:stark-versions"
 	},
 	"git": {
+		"changelog": "npm run generate:changelog-recent",
```

Comment:
Why did you move the "changelog" script in "git" instead of let it in "scripts" ?

Reply:
Because of: https://github.com/release-it/release-it/commit/b2f0fc2a0459a5f670fdb260d57432033326c588

Reply:
Oops. I took a look on the releases but I didn't see it ^^